### PR TITLE
 - adds int16 format

### PR DESCRIPTION
**Pull request by baywet**
related #3167

### CODE REVIEW

**Comment by handrews** [comment]
@OAI/tsc was there any particular reason `int16` wasn't included in the first place?

File: registries/_format/int16.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+layout: default
+---
+
+# <a href="..">{{ page.collection }}</a>
+
+## {{ page.slug }} - {{ page.description }}
+
+Base type: `{{ page.base_type }}`.
+
+The `{{page.slug}}` format represents a signed 16-bit integer, with the range -32768 through +32767. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
```
Comment (baywet) [inline registries/_format/int16.md:15]:
```suggestion
The `{{page.slug}}` format represents a signed 16-bit integer, with the range -32768 through +32767.
```

**Comment by baywet** [comment]
@darrelmiller I believe this one is good to merge as we've been through all the required updates.

**Review by MikeRalphson — APPROVED** [review]
LGTM

**Pull request merged**